CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 ### 4.0.0 (Next)
+* [#176](https://github.com/alexa-js/alexa-app/pull/176): Added flexibility to attach alexa-app to express app or router - [@bgjehu](https://github.com/bgjehu).
```

Comment:
First, I just want to say that I didn't write this code, the latest iteration was @mreinstein who should comment. Now that I am paying attention I can see what you're saying, but lets try to make some sense out of it. 

A router or an app are the same thing, a Router instance is a complete middleware and routing system (or mini-app as express calls it). This code mounts the alexa app inside the router (or app) given to it. But then we still need to attach the new router to the express app, don't we? Which is the first thing done in the code:

```
var endpoint = "/" + options.endpoint;
expressApp.use(endpoint, router);
```